action/flow/model/simple/iteratorbehavior.go

```diff
@@ -44,6 +45,8 @@ func (tb *IteratorTaskBehavior) Eval(ctx model.TaskContext) (evalResult model.Ev
 		case string:
 			count, err := data.CoerceToInteger(iterateOn)
 			if err != nil {
+				err = fmt.Errorf("unsupported string %s for iterator", iterateOn)
```

Comment:
might want to use '%s' in case the string has spaces so the user can clearer see it